data_structures/src/data_request.rs

```diff
@@ -343,38 +343,39 @@ pub fn create_tally<RT>(
     dr_output: &DataRequestOutput,
     pkh: PublicKeyHash,
     report: &RadonReport<RT>,
-    reveals: Vec<RevealTransaction>,
-    n_commits: usize,
+    reveal_pkhs: Vec<PublicKeyHash>,
+    commit_pkhs: Vec<PublicKeyHash>,
```

Comment:
Maybe this is more self-descriptive and less redundant:
```suggestion
    revealers: Vec<PublicKeyHash>,
    commiters: Vec<PublicKeyHash>,
```